Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitFacadeHandler.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Facades
```

Comment:
Do we need a new namespace for this?

Reply:
I think so. I want to keep facades sort of cordoned off since they only live in the editor.

Reply:
@davidkline-ms I know we're trying to keep namespaces to a minimum, what's your take here?